test/cpp/end2end/xds/xds_wrr_end2end_test.cc

```diff
@@ -40,12 +42,25 @@ using ::envoy::extensions::load_balancing_policies::
     client_side_weighted_round_robin::v3::ClientSideWeightedRoundRobin;
 using ::envoy::extensions::load_balancing_policies::wrr_locality::v3::
     WrrLocality;
-using WrrTest = XdsEnd2endTest;
+
+class WrrTest : public XdsEnd2endTest {
+ protected:
+  void SetUp() override {
```

Comment:
what happens if in the future we put more stuff in `XdsEnd2endTest::SetUp`

Reply:
This is an already-established pattern; we already have a bunch of places where subclasses override `SetUp()` so that individual tests can call `InitClient()` themselves.  We originally moved code from `SetUp()` to `InitClient()` for the specific purpose of supporting cases like this.  If we need to add more code in the future, we would add it in `InitClient()` instead of in `SetUp()`.